setup.py

```diff
@@ -49,6 +52,24 @@
     version = re.search(r'^__version__\s*=\s*[\'"]([^\'"]*)[\'"]',
                         fd.read(), re.MULTILINE).group(1)
 
+class TestCommand(Command):
+
+    user_options = [ ]
+    __dir__ = os.path.dirname(os.path.realpath(__file__))
+
+    def initialize_options(self):
+        os.chdir(os.path.join(self.__dir__, 'tests'))
+
+    def finalize_options(self):
+        pass
+
+    def run(self):
+        test_suite = unittest.TestLoader().discover('.', pattern='test*.py')
+        result = unittest.TextTestRunner(buffer=True).run(test_suite)
+        sys.exit(not result.wasSuccessful())
+
+cmdclass = {'test':TestCommand}
```

Comment:
This doesn't follow PEP8 specification.

Reply:
Fixed. I also fixed some other issues reported by flake8.